File: Back/src/Eventos.Domain/Palestrante.cs
using System.Collections.Generic;

namespace Eventos.Domain
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sobre { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<RedeSocial> RedeSociais { get; set; }

        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: Front/EventosWeb-App/src/app/Models/Palestrante.ts
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";

export interface Palestrante {
  id: number;
  nome: string;
  sobre: string;
  imagemURL: string;
  telefone: string;
  email: string;
  redeSocial: RedeSocial[];
  palestranteEvento: Evento[];
}
